01_Prototype/권우희/Prototype.md

```
@@ -3,32 +1,27 @@
 ## 목적
-
 - 객체 생성 비용이 큰 경우, 프로토타입을 사용해 빠르고 효율적으로 객체를 복제할 수 있다.
 - 객체 복사본을 생성할 때 원본 객체와 동일한 상태를 유지하도록 한다.
 - 복사된 객체는 원본과 독립적으로 사용될 수 있다.
 
 ## 단점
-
 - 복제 과정의 복잡성
-  - 객체 내부에 참조형 필드가 많거나 복잡한 구조일 경우, 깊은 복사를 제대로 구현하기 어렵고 까다롭다.
-  - 특히, 다중 중첩된 참조 객체를 모두 복제해야 할 때 코드가 복잡해지고 유지보수가 어려워질 수 있다.
+    - 객체 내부에 참조형 필드가 많거나 복잡한 구조일 경우, 깊은 복사를 제대로 구현하기 어렵고 까다롭다.
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)